CoreFoundation/String.subproj/CFString.c

```diff
@@ -1476,16 +1476,17 @@ CF_PRIVATE CFStringRef __CFStringCreateImmutableFunnel3(
                 useNullByte = hasNullByte;
 
             } else {	// Inline data; reserve space for it
-
```

Comment:
Again, please don't make unrelated whitespace changes. After reverting this change, please squash the commit.